src/reference/snakemake-style-guide.rst

```diff
@@ -220,9 +221,11 @@ Example:
        "logs/filter.txt"
    shell:
        r"""
+       exec &> >(tee {log:q})
+
        augur filter \
            --metadata {input.metadata:q} \
-           --output-metadata {output.metadata:q} 2>&1 | tee {log:q}
+           --output-metadata {output.metadata:q}
        """
 
 Before using ``tee``, ensure that your workflow uses `bash's pipefail option <http://redsymbol.net/articles/unofficial-bash-strict-mode/>`_, so successful ``tee`` execution does not mask errors from earlier commands in the pipe.
```

Comment:
I believe this paragraph about `pipefail` no longer applies with the updated `exec` recommendation.